conceptpower/classes.py

```diff
@@ -157,4 +159,65 @@ def get_type(self, uri):
                 data['supertype_id'] = snode.get('supertype_id')
                 data['supertype_uri'] = snode.get('supertype_uri')
         return data
-        
\ No newline at end of file
+
+    def create(self, user, password, word, pos, conceptlist, description, type, synonym_ids, equal_uris, similar_uris):
```

Comment:
Make sure that this line doesn't exceed 79 characters -- it can span multiple lines, if needed. e.g

```python
def create(self, user, password, word, pos, conceptlist, description, type, 
           synonym_ids, equal_uris, similar_uris):
```